app/javascript/packs/public.js

```diff
@@ -280,6 +280,17 @@ function main() {
       target.style.display = 'block';
     }
   });
+
+  // Empty the honeypot fields in JS in case something like an extension
+  // automatically filled them.
+  delegate(document, '#registration_new_user,#new_user', 'submit', () => {
```

Comment:
Is this not going to help bots that execute JS, i.e. headless Chrome?

Reply:
It probably is… I don't know how prevalent those are.

Reply:
I suspect those that execute JS would ignore fields with `display: none` anyway